pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,117 @@
+# Test ownership
+Test ownership is critical to adopting a healthy testing culture at your organization. By defining test owners, your teams will become accountable for maintaining a fast and reliable test suite, ensuring confidence when you deploy your code.
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
+
+Test ownership is managed via team assignments in a TESTOWNER file. The team that is the default owner of a test [will be automatically assigned flaky tests](/docs/test-analytics/flaky-test-assignment) to triage.
+
+> 🚧 Buildkite test ownership is currently in private beta
+> Please reach out to our support team to register for early access.
+
+## Setting test ownership
+
+You can upload a TESTOWNER file via this API endpoint:
+
+```bash
+curl --location 'https://analytics-api.buildkite.com/v1/test-ownerships' \
+     --header "Authorization: Bearer <your-suite-api-token>" \
+     -F 'file=@<your-TESTOWNER-file-location>'
+```
+You might consider creating a new pipeline to automatically upload your TESTOWNER file when changes are detected.
+
+## Example TESTOWNER file
+
+```bash
+# This is a comment.
+# Each line is a file pattern followed by one or more team slugs.
+
+# These teams will be the owners for everything in
+# the repo. While both teams own the test, the first team listed will
+# become the default owner for tests in this directory. Unless
+# a later match takes precedence, team-slug-1 will have any
+# flaky tests assigned to them by default.
+*       team-slug-1 team-slug-2
```

Comment:
Minor tweaks...

```suggestion
# These teams will be the test owners for everything in this
# repository. While both teams own the test, the first team listed will
# become the default owner for tests in this directory. Unless
# a later match takes precedence, team-slug-1 will have any
# flaky tests assigned to them by default.
* team-slug-1 team-slug-2
```

Also, couple of queries:

1. What was the reason for the extra space between the `*` and `team-slug-1`? If this functionally does nothing, I'd suggest removing it (as I've suggested above). Otherwise, please feel free to add it back in, with an explanation as to why the extra space is important.
2. When you say "a later match takes precedence", what exactly is being matched here and by what? (Sorry for my ignorance.)

Reply:
The weird spacing has come from me copying over [Github's instructions for creating a CODEOWNER file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) and overwriting it to match what we have. TESTOWNERSHIP uses the same rules as those set out for githubs `.gitignore` and `CODEOWNER` files, and I wanted to reuse something familiar. My assumption re the spacing is to try and show the difference between the location and the owner slugs. It's also how this [kind of file is usually laid out in real life](https://github.com/buildkite/buildkite/blob/main/.github/CODEOWNERS), but agree spacing is inconsistent e.g.
```bash
/docs docs docs-1
# as opposed to
/docs          docs docs-1
```

> When you say "a later match takes precedence", what exactly is being matched here and by what?

I think I answered this in https://github.com/buildkite/docs/pull/2813#discussion_r1641989485 by let me know if it's still unclear!

Reply:
Thanks! Fixed and reworded following a more detailed discussion with you :-)